compiler/daml-licenses/licenses/zlib.cabal

```diff
@@ -8,8 +8,8 @@ maintainer:      Duncan Coutts <duncan@community.haskell.org>
 bug-reports:     https://github.com/haskell/zlib/issues
 category:        Codec
 synopsis:        Compression and decompression in the gzip and zlib formats
-description:     This package provides a pure interface for compressing and 
```

Comment:
While those files should go completely, it seems somewhat questionable to modify them ourselves.

Reply:
Good point. I must confess I did not spend much time looking at each individual file. I'll revert that one.